swap_meet/decor.py

```diff
@@ -1,2 +1,10 @@
-class Decor:
-    pass
+from swap_meet.item import Item
+
+class Decor(Item):
+    
+    def __init__(self, category = "Decor", condition = None, age = None):
+        super().__init__(category, condition, age)
```

Comment:
great job on using inheritance here. I see that you've added age here but I don't see it being used in your code. What would you want to use it for?